homeassistant/components/notify/hangouts.py

```diff
@@ -0,0 +1,76 @@
+"""
```

Comment:
For simplicity, let's move this platform to a new PR.

Reply:
sure? notify.hangouts needs the hangouts component to work and on the big rework on the hangouts component i also changes the notify component